chanzuckerberg/idseq-web
# Prepare to serve static assets via Cloudfront

# Description

This makes idseq rails app use a new env var `CLOUDFRONT_ENDPOINT` if available on init .

The future values should be `assets.staging.idseq.net` and `assets.idseq.net`. 

# Notes

I followed the guides https://aws.amazon.com/blogs/developer/caching-the-rails-asset-pipeline-with-amazon-cloudfront/ and https://aws.amazon.com/premiumsupport/knowledge-center/cloudfront-distribution-serve-content/

# Tests

* start locally without `CLOUDFRONT_ENDPOINT`, see assets served locally
* set `CLOUDFRONT_ENDPOINT` to `assets.staging.idseq.net`, see assets served remotely, see site load correctly
